packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -474,9 +477,10 @@ class CupertinoTextField extends StatefulWidget {
   /// If set, a character counter will be displayed below the
   /// field, showing how many characters have been entered and how many are
   /// allowed. After [maxLength] characters have been input, additional input
-  /// is ignored, unless [maxLengthEnforced] is set to false. The TextField
-  /// enforces the length with a [LengthLimitingTextInputFormatter], which is
-  /// evaluated after the supplied [inputFormatters], if any.
+  /// is ignored, unless [maxLengthEnforced] is set to false or
```

Comment:
Master channel failed to run a demo app on my iOS simulator, can you help me with a GIF for this behavior?

Reply:
I saw it ignores the limit if `false`, the only difference is without warning. Something I have missing?

Reply:
Oh I'm kind of understand what @LongCatIsLooong is talking about. If `maxLengthEnforced` is `false`, it equals to `maxLength: null`, so there is no need for this parameter. Can we remove it directly without deprecation? @justinmc

Reply:
We could try, but if anyone is passing it in it will break them, even if their functionality doesn't change.  It will be simpler just to wait until maxLengthEnforced is removed everywhere, but it's up to you if you want to try.

Reply:
Decide not to remove currently and wait for deprecation ends.